### ISSUE TITLE
Dark mode toggle is not reachable with keyboard navigation

### ISSUE THREAD

**Issue by EmmaDawsonDev** [EddieHubCommunity/LinkFree#1927]
### Description

<img width="74" alt="image" src="https://user-images.githubusercontent.com/57045550/194723734-c74a9280-e4cd-4c3b-a27e-98f058311116.png">

The toggle button in the header is unreachable with the tab key when navigating by keyboard. It should be possible to reach it and activate it just by tabbing through the site.

**Comment by vasucp1207** [comment]
Hey, @EmmaDawsonDev I want to work on this issue

**Comment by vasucp1207** [comment]
Hey, @EmmaDawsonDev PR done take a look at that.

**Comment by eddiejaoude** [comment]
The new version of LinkFree is out, I will close this issue, feel free to create a new issue for the new version if still relevant
